core/src/main/java/com/novoda/noplayer/internal/mediaplayer/AndroidMediaPlayerFacade.java

```diff
@@ -191,14 +191,14 @@ int getDuration() throws IllegalStateException {
         return mediaPlayer.getDuration();
     }
 
-    int getCurrentPosition() throws IllegalStateException {
+    int currentPositionInMillis() throws IllegalStateException {
```

Comment:
interesting that theirs is an int... I guess that is fairly big in Java. Traditional to use long for video timestamps but I guess in this context int will cover up to almost 600 hours so is sufficient. Should we use the same for consistency, to avoid confusion? Or better to go with more of a typical value for video things?

Reply:
I'd go with the more typical, always found it a bit strange that they went with `int` it would work in most cases I guess but 🤷‍♂️